vips/vip-444/bscmainnet.ts

```diff
@@ -0,0 +1,198 @@
+      },
+      {
+        target: market.vToken.address,
+        signature: "setReserveFactor(uint256)",
+        params: [market.riskParameters.reserveFactor],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [market.vToken.underlying.address, market.initialSupply.amount, NORMAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: market.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, market.initialSupply.amount],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            market.vToken.address,
+            market.riskParameters.collateralFactor,
+            market.riskParameters.liquidationThreshold,
+            market.initialSupply.amount,
+            market.initialSupply.vTokenReceiver,
+            market.riskParameters.supplyCap,
+            market.riskParameters.borrowCap,
+          ],
+        ],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: market.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, 0],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: COMPTROLLER_CORE,
+        signature: "setActionsPaused(address[],uint8[],bool)",
+        params: [[market.vToken.address], [2], true],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: USDT_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[0], [token.address], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: USDC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[1], [token.address], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: WBTC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[2], [token.address], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: WETH_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[3], [token.address], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: XVS_VAULT_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[4], [token.address], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.ethereum,
+      },
```

Comment:
The conversions of weETHs are already configured in the converters. We don't need to re-configure them

Reply:
https://github.com/VenusProtocol/vips/pull/484/commits/abc5bde91b713fd5fedc9b3c20fde55de85fef70